src/js/options.js

```diff
@@ -562,6 +596,9 @@ function addOrigins(e) {
   // activate tooltips
   $('#blockedResourcesInner .tooltip:not(.tooltipstered)').tooltipster(
     htmlUtils.DOMAIN_TOOLTIP_CONF);
+
+  // register the newly-created toggle switch so that user changes are saved
+  registerToggleHandlers($(target).children().last().find('.switch-toggle'));
```

Comment:
Does this double-register events (for non-new toggles)?

Reply:
It wasn't meant to, but I see now that it does sometimes on scrolling up and down! Will fix